runtime/oti/vm_api.h

```diff
@@ -1422,6 +1422,19 @@ exitJavaVM(J9VMThread * vmThread, IDATA rc);
 IDATA
 findArgInVMArgs(J9PortLibrary *portLibrary, J9VMInitArgs* j9vm_args, UDATA match, const char* optionName, const char* optionValue, UDATA doConsumeArgs);
 
+/**
+* @brief
+* @param j9vm_args
+* @param match
+* @param optionName
+* @param optionValue
+* @param doConsumeArgs
+* @param doConsumeIgnoredArgs
+* @return IDATA
```

Comment:
Can you fill in the values for these doc annotations?  I know existing code hasn't done a great job of actually provide descriptions of the function / arguments / return value but let's try to do better =)